test/test_helper.rb

```diff
@@ -5,6 +5,7 @@
 class ActiveSupport::TestCase
   # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
   fixtures :all
-
```

Comment:
I did not delete any line. just added Devise::TestHelpers to pass all test cases.